tuxemon/core/states/combat/combat.py

```diff
@@ -643,7 +643,7 @@ def perform_action(self, user, technique, target=None):
             message = ''
 
         try:
-            tools.load_sound(technique.sfx).play()
+            tools.load_sound(db.lookup_file("sounds", technique.sfx)).play()
```

Comment:
`tools.load_sound` is already supposed to take the pain out of path/file management.  consider refactoring this to accept a slug to make it more ergonomic.

Reply:
👍 i'll do some thinking and see what i can come up with